specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/subscriptions.json

```diff
@@ -134,6 +234,68 @@
         }
       },
       "x-ms-azure-resource": true
+    },
+    "OperationResult": {
```

Comment:
OperationResult is a generic concept. It's not specific to a certain API. The response for query testing api should have its own specific definition/object model